Makefile

```
@@ -0,0 +1,35 @@
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+MAKEFLAGS += --warn-undefined-variables
+.DEFAULT_GOAL := help
+
+PWD := $(realpath $(dir $(abspath $(firstword $(MAKEFILE_LIST)))))
+
+PY := python
+PIP := $(PY) -m pip
+
+import_check: ## Check if the package can be imported.
+	$(PY) -c 'import pytorch_pfn_extras'
+
+format: ## Format the Python code.	
```

Nit: I like the style of putting `.PHONY:` just before the rule for each target (I learned this from imos-san). This will make maintenance easier when adding/removing targets.

```suggestion
.PHONY: format
format: ## Format the Python code.	
```

---

I have followed that format.